client/src/test/diagnostics.test.ts

```diff
@@ -26,83 +26,76 @@ describe('Should get diagnostics for a valid empty ⚡ page', () => {
         severity: vscode.DiagnosticSeverity.Error,
         source: '⚡',
         code: 'MANDATORY_TAG_MISSING',
       },
       {
         message:
-          "⚡ : The mandatory tag 'body' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml#required-markup",
+          "⚡ : The mandatory tag 'body' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#required-markup",
         range: toRange(17, 7, 17, 12),
         severity: vscode.DiagnosticSeverity.Error,
         source: '⚡',
         code: 'MANDATORY_TAG_MISSING',
       },
       {
         message:
-          "⚡ : The mandatory tag 'noscript enclosure for boilerplate' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amp-boilerplate?format=websites",
+          "⚡ : The mandatory tag 'noscript enclosure for boilerplate' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amp-boilerplate/?format=websites",
         range: toRange(17, 7, 17, 12),
         severity: vscode.DiagnosticSeverity.Error,
         source: '⚡',
         code: 'MANDATORY_TAG_MISSING',
       },
       {
         message:
-          "⚡ : The mandatory tag 'head > style[amp-boilerplate]' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amp-boilerplate?format=websites",
+          "⚡ : The mandatory tag 'head > style[amp-boilerplate]' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amp-boilerplate/?format=websites",
         range: toRange(17, 7, 17, 12),
         severity: vscode.DiagnosticSeverity.Error,
         source: '⚡',
         code: 'MANDATORY_TAG_MISSING',
       },
       {
         message:
-          "⚡ : The mandatory tag 'noscript > style[amp-boilerplate]' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amp-boilerplate?format=websites",
+          "⚡ : The mandatory tag 'noscript > style[amp-boilerplate]' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amp-boilerplate/?format=websites",
         range: toRange(17, 7, 17, 12),
         severity: vscode.DiagnosticSeverity.Error,
         source: '⚡',
         code: 'MANDATORY_TAG_MISSING',
       },
       {
         message:
-          "⚡ : The mandatory tag 'amphtml engine v0.js script' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml#required-markup",
+          "⚡ : The mandatory tag 'amphtml engine script' is missing or incorrect. : https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#required-markup",
         range: toRange(17, 7, 17, 12),
         severity: vscode.DiagnosticSeverity.Error,
         source: '⚡',
         code: 'MANDATORY_TAG_MISSING',
       },
-      {
-        message: 'Doctype must be declared first.',
```

Comment:
Why is this removed?

Reply:
This error is false positive, because the input file [amp-empty.html](https://github.com/diparikh-project/vscode-amphtml-validator/blob/6628ff875886d54a8b918d82880b54491e932f34/client/testFixture/amp/amp-empty.html) does declare the `Doctype`.
This is removed because the latest validator no longer generate this error.